fieldMapping.json

```diff
@@ -0,0 +1,212 @@
+        }
+      ]
+    },
+    {
+      "Name": "JobTitleCode",
+      "Type": "Text",
+      "MappingActions": [
+        {
+          "MapForActions": ["Create", "Update"],
+          "MappingMode": "Field",
+          "Value": "\"Person.PrimaryContract.Title.ExternalId\"",
+          "UsedInNotifications": false,
+          "StoreInAccountData": false
+        }
+      ]
+    },
+    {
+      "Name": "JobTitle",
+      "Description": "",
+      "Type": "Text",
+      "MappingActions": [
+        {
+          "MapForActions": ["Create", "Update"],
+          "MappingMode": "Field",
+          "Value": "\"Person.PrimaryContract.Title.Name\"",
+          "UsedInNotifications": false,
+          "StoreInAccountData": true
+        }
+      ]
+    },
+    {
+      "Name": "NamePrefix",
+      "Type": "Text",
+      "MappingActions": [
+        {
+          "MapForActions": ["Create", "Update"],
+          "MappingMode": "Complex",
+          "Value": "\"function getPrefixes() {\\n    let middleName = Person.Name.FamilyNamePrefix;\\n    let middleNamePartner = Person.Name.FamilyNamePartnerPrefix;\\n    let convention = Person.Name.Convention;\\n\\n    let nameFormatted = '';\\n    let prefix = '';\\n\\n    switch (convention) {\\n        case \\\"P\\\":\\n        case \\\"PB\\\":\\n            nameFormatted = middleNamePartner;\\n            break;\\n        case \\\"B\\\":\\n        case \\\"BP\\\":\\n        default:\\n            nameFormatted = middleName;\\n            break;\\n    }\\n\\n    if (typeof nameFormatted !== 'undefined' && nameFormatted) {\\n        prefix = nameFormatted.trim();\\n    } else {\\n        prefix = nameFormatted;\\n    }\\n\\n    return prefix;\\n}\\n\\ngetPrefixes();\"",
+          "UsedInNotifications": false,
+          "StoreInAccountData": false
+        }
+      ]
+    },
+    {
+      "Name": "Active",
+      "Type": "Text",
+      "MappingActions": [
+        {
+          "MapForActions": ["Enable"],
+          "MappingMode": "Fixed",
+          "Value": "\"1\"",
+          "UsedInNotifications": false,
+          "StoreInAccountData": false
+        },
+        {
+          "MapForActions": ["Disable", "Create"],
+          "MappingMode": "Fixed",
+          "Value": "\"0\"",
+          "UsedInNotifications": false,
+          "StoreInAccountData": false
+        }
+      ]
+    },
+    {
+      "Name": "BusinessMail",
+      "Description": "",
+      "Type": "Text",
+      "MappingActions": [
+        {
+          "MapForActions": ["Create", "Update"],
+          "MappingMode": "Complex",
+          "Value": "\"function getMailAddress() {\\r\\n    let mail = '';\\r\\n\\r\\n    if (typeof Person.Accounts._bccd667807a9431a9d4acfc64c19a3d6.mail !== 'undefined' && Person.Accounts._bccd667807a9431a9d4acfc64c19a3d6.mail) {\\r\\n        mail = Person.Accounts._bccd667807a9431a9d4acfc64c19a3d6.mail;\\r\\n    }\\r\\n\\r\\n    return mail;\\r\\n}\\r\\n\\r\\ngetMailAddress()\"",
```

Comment:
This GUID should probably be changed to the name of the system.

Reply:
Replaced the guid with the systemname